### DISCUSSION TITLE
Casting to a TV/Chromecast/Airplay?

### DISCUSSION THREAD

**Discussion by JoshuaPickett** [KRTirtho/spotube#1451] [Ideas]
It would be awesome to be able to Cast to a TV/Chromecast/Airplay

**Comment by Sowden** [comment]
Yeah, actually I'm looking on how to play the app on a device as well.  Now I get that there isn't a spototube app on my Chromcast TV.  But can it play back on its normal generic player?  I know that I can (locally) play my videos on my NAS through their local player.  Could spototube do the same?

**Reply by characta** [reply]
@Sowden Did you get any further?

**Reply by Sowden** [reply]
I have sort of found a solution, but its not a the gold standard.  I found that I can cast the screen of my phone to my chromcast on my TV.  But instead of casting my whole phone, at least on my Android 14 motorola Razor, I can select to cast a certain app, so that my whole phone isn't on the screen at all times.  This has varying results, as the music skips around a lot, like a poorly downloaded MP3.  But its not the solution that we need.  I do know that apps like Twitch and my Criterion app all cast to my chromecast using a generic casting app on the chromcast, just like I mentioned above that my videos on my local NAS play the same way.  So I am wondering why Spotube can't do the same thing.  If anyone on here would like to chime in, I would love to hear some perspectives on this.

**Comment by mvk-wayne** [comment]
Is it technically possible?

**Comment by characta** [comment]
@KRTirtho I'd love to get it going with my HifiBerry. So being able to use either AirPlay, DLNA, OpenHome or [Snapcast](https://github.com/badaix/snapcast?tab=readme-ov-file) would be great :) 
Would love to hear your thoughts and how I can help.
